lib/Varien/Db/Tree.php

```diff
@@ -65,11 +65,6 @@ public function __construct($config = [])
             // convenience variable
             $db = $config['db'];
 
-            // use an object from the registry?
-            if (is_string($db)) {
-                $db = Zend::registry($db);
-            }
-
```

Comment:
since Zend class doesn't exist.. this code was never called so it can also be removed